web/src/services/cipher_keys.ts

```
@@ -0,0 +1,67 @@
+          iv: string;
+          encrypted_data: string;
+        };
+      };
+    };
+  };
+}
+
+interface CipherKeyData {
+  name: string;
+  key: AkeylessStore;
+  provider: Object;
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)